widget/ui/src/components/StepDetails/StepDetails.tsx

```diff
@@ -51,27 +65,64 @@ const StepDetailsComponent = forwardRef<HTMLDivElement, StepDetailsProps>(
         state={state}
         ref={containerRef}
         tabIndex={tabIndex}>
-        <div className={swapperStyles()}>
-          {type === 'swap-progress' && hasSeparator && (
-            <StepSeparator state={state} />
-          )}
-          <SwapperImage state={state}>
-            <Image size={22} src={step.swapper.image} />
-          </SwapperImage>
-          <Typography
-            className="swapper__description"
-            ml={8}
-            size="small"
-            variant="body">
-            {i18n.t({
-              id: 'Swap on {fromChain} via {swapper}',
-              values: {
-                fromChain: step.from.chain.displayName,
-                swapper: step.swapper.displayName,
-              },
+        {type === 'quote-details' && (
+          <div className={swapperItemStyles()}>
+            <SwapperImage state={state}>
+              <Image size={22} src={step.swapper.image} />
+            </SwapperImage>
+            <Typography ml={8} size="medium" variant="label">
+              {i18n.t({
+                id: 'Swap on {fromChain} via {swapper}',
+                values: {
+                  fromChain: step.from.chain.displayName,
+                  swapper: step.swapper.displayName,
+                },
+              })}
+            </Typography>
+          </div>
+        )}
+
+        {type === 'swap-progress' && (
+          <div className={swappersStyles()}>
+            {hasSeparator && <StepSeparator state={state} />}
+            {swappers.map((swapperItem, index) => {
+              const key = `${swapperItem.swapper.displayName}-${index}`;
+              return (
+                <>
```

Comment:
I think you need to put key for the parent element.
`<React.Fragment key={key}>`

Reply:
Fixed